components/Workout/Workout.js

```diff
@@ -67,13 +72,14 @@ export default function Workout({ workout, handleDelete, handleEditWorkout }) {
             </ModalContent>
           </ModalOverlay>
         ) : null}
```

Comment:
```suggestion
        ) : null}
```
see above. you could use `{deleteMode && (...)}` withour `:null`